src/components/People.tsx

```diff
@@ -61,62 +62,7 @@ const People: React.FC = () => {
-              <Member
-                name="Liam Moynihan"
-                position="President"
-                image="https://c4cneu-public.s3.us-east-2.amazonaws.com/Board/2020/Liam-M.jpg"
-                linkedin="https://www.linkedin.com/in/liam-moynihan-b75806123/"
-                email="moynihan.li@northeastern.edu"
-              />
-              <Member
-                name="Jack Blanc"
-                position="Director of Engineering"
-                image="https://c4cneu-public.s3.us-east-2.amazonaws.com/Board/2020/Jack-B.jpg"
-                linkedin="https://www.linkedin.com/in/blancjack/"
-                email="blanc.ja@northeastern.edu"
-              />
-              <Member
-                name="Reine Nisheiwat"
-                position="Director of Design"
-                image="https://c4cneu-public.s3.us-east-2.amazonaws.com/Board/2020/Reine-N.jpg"
-                linkedin="https://www.linkedin.com/in/reinenish/"
-                email="nisheiwat.r@northeastern.edu"
-              />
-              <Member
-                name="Sadaf Khansalar"
-                position="Administrative Director"
-                image="https://c4cneu-public.s3.us-east-2.amazonaws.com/Board/2020/Sadaf-K.jpg"
-                linkedin="https://www.linkedin.com/in/sadafkhansalar/"
-                email="khansalar.s@northeastern.edu"
-              />
-              <Member
-                name="Abby Emison"
-                position="Recruitment & Community Engagement Director"
-                image="https://c4cneu-public.s3.us-east-2.amazonaws.com/Board/2020/Abby-E.jpg"
-                linkedin="https://www.linkedin.com/in/abigail-emison-26b448125/"
-                email="emison.a@northeastern.edu"
-              />
-              <Member
-                name="Vedanshi Shah"
-                position="Marketing Director"
-                image="https://c4cneu-public.s3.us-east-2.amazonaws.com/Board/2020/Vedansha-S.jpg"
-                linkedin="https://www.linkedin.com/in/vedanshi-shah-267369179/"
-                email="shah.ve@northeastern.edu"
-              />
-              <Member
-                name="Jack Tonina"
-                position="School Liaison"
-                image="https://c4cneu-public.s3.us-east-2.amazonaws.com/Board/2020/Jack-T.jpg"
-                linkedin="https://www.linkedin.com/in/jack-tonina-6328b9147/"
-                email="tonina.j@northeastern.edu"
-              />
-              <Member
-                name="Justin Konecny"
-                position="Technical Lead"
-                image="https://c4cneu-public.s3.us-east-2.amazonaws.com/Board/2020/Justin-K.jpg"
-                linkedin="https://www.linkedin.com/in/justin-konecny/"
-                email="konecny.j@northeastern.edu"
-              />
+              {PeopleData.map(person => <Member name={person.name} position={person.position} image={person.image} linkedin={person.linkedin} email={person.email} />)}
```

Comment:
The `Member` component should take as props a `Person`.
You can do `<Member person={person} />` (might have to give them different names)